nion/swift/model/DisplayItem.py

```diff
@@ -3298,8 +3298,10 @@ def data_info(self) -> DisplayItem.DataInfo:
         calibrated_dimensional_calibrations = self.calibrated_dimensional_calibrations
         calibrated_dimensional_calibration_str_list = list[str]()
         for index, dimensional_calibration in enumerate(calibrated_dimensional_calibrations or list()):
-            converter = CalibratedSizeFloatToStringConverter(self, index, uniform=False)
-            calibrated_dimensional_calibration_str_list.append(converter.convert_calibrated_value_to_str(display_data_shape[index]))
+            converter = CalibratedSizeFloatToStringConverter(self, index, uniform=True)
+            calibrated_value = converter.convert(1)
+            if calibrated_value is not None:
+                calibrated_dimensional_calibration_str_list.append(calibrated_value) #passing in 1 as the converter expects fractional units
```

Comment:
Why is this being changed? This belongs in a separate PR with an explanation.

Also, the format of the comment would need to follow Python conventions with two spaces before the '#' and a space after the '#'.

Reply:
See related PR https://github.com/nion-software/nionswift/pull/1214